src/statusUpdate.js

```diff
@@ -0,0 +1,7 @@
+function statusUpdate(e, i) {
+  const savedList = JSON.parse(localStorage.getItem('savedList'));
```

Comment:
- [ ] I would recommend using ES6 syntax. Use arrow function instead of `function` keyword to make the code cleaner.